sei/chain.json

```diff
@@ -0,0 +1,118 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "sei",
+    "status": "live",
+    "website": "https://www.sei.io/",
+    "network_type": "mainnet",
+    "pretty_name": "Sei",
+    "chain_id": "pacific-1",
+    "bech32_prefix": "sei",
+    "daemon_name": "seid",
+    "node_home": "$HOME/.sei",
+    "key_algos": [
+        "secp256k1"
+    ],
+    "slip44": 118,
+    "fees": {
+        "fee_tokens": [
+            {
+                "denom": "usei",
+                "fixed_min_gas_price": 0,
```

Comment:
pacific-1 (sei mainnet) has a global min gas fee of `0.1` as of a recent gov prop. Additionally, if this value changes up or down in the future, we should update this via PR, right?

Reply:
Is this mean to serve as a source of truth for gas prices, or just a guidance? curious how strictly these values need to match the actual chain parameters